### PR TITLE
[examples] Add an example that shows how to switch styles

### PR DESCRIPTION
**Pull request by jmkiley**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-ios/blob/main/CONTRIBUTING.md#contributor-license-agreement).

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [x] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [x] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-ios` changelog: `<changelog></changelog>`.
 - [ ] Update the migration guide, API Docs, Markdown files - Readme, Developing, etc

### Summary of changes

This PR adapts the pre-v10 example that shows how to switch styles. The new example includes how to update the `StyleURI` to a custom style and includes comments to show how to use a local style JSON.

cc @Chaoba

### CODE REVIEW

**Comment by jmkiley** [comment]
Will update this to include six options.

**Comment by jmkiley** [comment]
Per chat with @Chaoba:
- Updated this example to use a local style JSON.
- Added a comment outlining other Mapbox default styles that can be used.

File: Apps/Examples/Examples/All Examples/OfflineRegionManagerExample.swift [hunk 1]
```diff
@@ -1,107 +0,0 @@
-import Foundation
```
Comment (neelmistry94) [inline Apps/Examples/Examples/All Examples/OfflineRegionManagerExample.swift:1]: why is the being deleted?

**Review by neelmistry94 — APPROVED** [review]
lgtm

**Pull request merged**